app/jobs/prepare_base_notifications_job.rb

```diff
@@ -38,6 +42,17 @@ def subscriptions(move, only_supplier_id: nil)
     Subscription.kept.enabled.where(supplier: suppliers)
   end
 
+  def build_notifications(subscription, type_id, topic, action_name)
+    notifications = []
+
+    if %w[update update_status].include?(action_name) &&
```

Comment:
I think we need to check that `topic` is move, right? otherwise we could create duplicate create notifications for other types

Reply:
Ah yes, fair point. Or should I move the logic into the PrepareMoveNotificationsJob?

Reply:
Yeah moving the logic into the PrepareMoveNotificationsJob makes more sense I think, good call

Reply:
ok I've moved some of the logic to the PrepareMoveNotificationsJob. It turns out that the other jobs that have `update` actions all also override the `prepare` method, so there wasn't any broken functionality to test for, but I've beefed up their tests a bit anyway to be safe.